# Generated by Django 4.1.7 on 2023-08-05 11:48

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('log_collector', '0009_taskscheduler_delete_cronscheduler'),
    ]

    operations = [
        migrations.AlterModelOptions(
            name='anylogger',
            options={'verbose_name': 'Any Logger', 'verbose_name_plural': 'Any Logger'},
        ),
        migrations.AlterModelOptions(
            name='collectorexception',
            options={'verbose_name': 'Self Collector Errors', 'verbose_name_plural': 'Self Collector Errors'},
        ),
        migrations.AlterModelOptions(
            name='djangoexception',
            options={'verbose_name': 'Captured Exceptions (Sentry like)', 'verbose_name_plural': 'Captured Exceptions (Sentry like)'},
        ),
        migrations.AlterModelOptions(
            name='djangologger',
            options={'verbose_name': 'Django Errors', 'verbose_name_plural': 'Django Errors'},
        ),
        migrations.AlterModelOptions(
            name='nginxlogger',
            options={'verbose_name': 'Nginx Logs', 'verbose_name_plural': 'Nginx Logs'},
        ),
        migrations.AlterModelOptions(
            name='taskscheduler',
            options={'verbose_name': 'Task Scheduler', 'verbose_name_plural': 'Task Scheduler'},
        ),
        migrations.AlterField(
            model_name='djangoexception',
            name='message',
            field=models.TextField(blank=True, null=True),
        ),
    ]
